doc.html

```diff
@@ -572,8 +572,81 @@ <h4 class="card-subheading">Sub Heading</h4>
 
         </div>
 <!----------------Cards component ends here-------------------------------->  
+  
+<!----------------Image component starts here-------------------------------->  
+        <div class="comp-container" id="Images">
+          <h1>Image</h1>
+          <p class="comp-decription">
+            Images can be responsive to fit the parent's width, and also can be customised to be round shaped.
+          </p>
+          <span class="example">Example of Responsive Image</span>
+          <p class="comp-example">
+            Copy paste the below code and replace the img url with your image url.
+          </p>
+          
+            <div class="comp-display-container">
+              <!----- main component code starts from here ----->
 
+              <div class="img-res-container">
+                <img class="img-responsive" src="./assets/userProfilePic.jpg" >
+              </div>
+                 <!------ image round ----->
+              <div class="img-res-container ">
+                <img class="img-responsive img-round" src="./assets/userProfilePic.jpg" >
+              </div>
 
+              <!----- main component code ends here ----->
+            </div>
+          
+          <div class="code_link">
+            <code>
+              <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%2867%2C196%2C255%2C1%29&t=one-light&wt=none&l=htmlmixed&width=500&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=165%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522img-res-container%2522%253E%250A%2520%2520%253Cimg%2520class%253D%2522img-responsive%2522%2520src%253D%2522.%252Fassets%252FuserProfilePic.jpg%2522%2520%253E%250A%253C%252Fdiv%253E%250A%2520%2520%2520%253C%21------%2520image%2520round%2520-----%253E%250A%253Cdiv%2520class%253D%2522img-res-container%2520%2522%253E%250A%2520%2520%253Cimg%2520class%253D%2522img-responsive%2520img-round%2522%2520src%253D%2522.%252Fassets%252FuserProfilePic.jpg%2522%2520%253E%250A%253C%252Fdiv%253E"
+              style="width:100%; height: 100%; border:0; transform: scale(1); overflow:hidden;"
+              sandbox="allow-scripts allow-same-origin">
+            </iframe>
+                
+            </code>
+          </div>
+        </div>
+<!----------------Image component ends here-------------------------------->  
+
+
+<!----------------Inputs component starts here-------------------------------->  
+<div class="comp-container" id="Inputs">
```

Comment:
Just a suggestion here that the indentation could be improved as the next `div` is not starting from where the above `div` has ended.